src/frontend/pages/home/home.css

```diff
@@ -10,31 +12,53 @@
     font-size: larger;
 }
 
+.title-main{
+    margin: 0rem auto;
+    text-align: center;
+    font-size:1.75rem;
+    padding:2rem;
+    font-weight: 600;
+    margin-top: 2rem;
+    color:var(--primary-color);
+}
+
+.main-home-body{
+    margin-top: 5rem;
+    background-color: var(--background-color);
+    min-height:100vh;
+}
+
 .categories-container{
-    padding:1rem 2rem ;
+    padding: 0 2rem;
     display: flex;
-    justify-content:space-evenly;
+    justify-content:center;
     align-items: center;
     flex-wrap: wrap;
     height: fit-content;
-    gap: 1rem;
+    gap:2.5rem;
 }
 
 .category-card{
     display: flex;
-    justify-content: center;
-    width:250px;
-    height: 250px;
-    flex-wrap: wrap;
+    flex-direction: column;
+    height:300px;
```

Comment:
Can use relative units instead of pixels for better result

Reply:
> Can use relative units instead of pixels for better result

yeah have used relative units only but only for card i m using absolute units. and handling width in media queries for better result. anyway i will try to implement using relative unit in future